src/components/input/input.scss

```diff
@@ -31,6 +31,15 @@ md-input-container {
     display: block;
   }
 
+  &.md-has-icon {
+    @include rtl(padding-left, $icon-offset, initial);
+    @include rtl(padding-right, initial, $icon-offset);
```

Comment:
I don't think `initial` is supported in IE.

http://caniuse.com/#feat=css-initial-value